.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
@feldlime @blondered There is an option to add like a contact-us-there thingy. If there is a corporate email address for RecTools, we can add it here. Or we can simply link the tg channel. It's up to us, just wanted to let you know.

Reply:
Nice option.

Just discussed today that our corporate email isn't really usable, so let's add ling to the tg channel

Reply:
Ok cool, sounds good to me.

Reply:
Added two things: the link to the docs, the tg channel link

Reply:
Great! I suggest to do with `https://rectools.readthedocs.io/` link for the docs. It will lead to the stable version of documentation. Not to the developer one

Reply:
Good catch, I've replaced the link